File: Euclid/Messages/Incoming/FINDUSER.cs
using Euclid.Game;
using Euclid.Network.Streams;

namespace Euclid.Messages.Incoming
{
    class FINDUSER : IMessageEvent
    {
        #region Properties

        public bool AuthenticationRequired => false;

        #endregion

        public void Handle(Player player, Request request)
        {

        }
    }
}

File: euclid/src/main/java/euclid/messages/incoming/FINDUSER.java
package euclid.messages.incoming;

import euclid.game.Player;
import euclid.messages.IMessageEvent;
import euclid.network.streams.Request;

public class FINDUSER implements IMessageEvent {

    @Override
    public boolean authenticationRequired() {
        return false;
    }

    @Override
    public void handle(Player player, Request request) {
    }
}
